keyboards/akko/top40/keymaps/vial/rules.mk

```diff
@@ -0,0 +1,15 @@
+VIA_ENABLE     = yes # Enable VIA
+VIAL_ENABLE    = yes # Enable VIAL
+VIALRGB_ENABLE = yes # Enable VIALRGB
+
+# Options to reduce firmware size:
+LTO_ENABLE = yes # make the compiler work harder
+
+# Vial optimization
+# Enable all as the WB32FQ95 has 256kB flash
+
+# QMK optimization
+# Enable all except mostly never used features
+SPACE_CADET_ENABLE = no
+GRAVE_ESC_ENABLE   = no
+MAGIC_ENABLE       = no
```

Comment:
```suggestion

```
Having these rules here are redundant due to the WB32FQ95 having 256KB of flash, which is already plenty.

Reply:
If there is plenty of space that does not mean we should waste it, I believe, so I turn on link time optimization for general size optimization and disable the features non-present in this particular keyboard. I would like to insist to leave this options in place even this chip offer a huge flash. What do you think?

Reply:
Ok, I agree, maybe to leave just LTO.